src/utils/proximity.ts

```diff
@@ -23,11 +13,7 @@ export async function getProximity(
     }
 
     cg: if (rule.type === "client-geolocation") {
```

Comment:
Would be nice to remove this weird label code, but I imagine it would be a pain in the backside, I just worry that it'll sit here until the next rewrite 😅